tests/testsuite/test.rs

```diff
@@ -4941,6 +4941,7 @@ fn cargo_test_set_out_dir_env_var() {
 
     p.cargo("test").run();
     p.cargo("test --package foo --test case -- tests::test_add --exact --nocapture")
+        .env_remove("OUT_DIR")
```

Comment:
Should we do this instead in https://github.com/rust-lang/cargo/blob/master/crates/cargo-test-support/src/lib.rs#L1249 ?

Reply:
Updated. Good suggestion. ❤️